pkg/util/ipconfig/ipconfig.go

```diff
@@ -42,7 +41,6 @@ const (
 
 // runner implements Interface in terms of exec("ipconfig").
 type runner struct {
-	mu   sync.Mutex
```

Comment:
this is fun, looks like this mutex was never used since the original impl PR introduced it?

Reply:
Probably the code was originally copied from some other directory. But it looks like no mutex is needed here.